### ISSUE TITLE
CTF: "tie" at top but "winner" on one player's side.

### ISSUE THREAD

**Issue by 7wells** [supertuxkart/stk-code#4347]
Hello,

In "Oliver's math class" track/arena, the kids played CTF. Both had the same amount of captured flags at the end, but one had more points. At the top, it read "tie", but on the side of one player, it read "winner". A quarrel began, if it now was a tie or if one was the winner _(sigh)_.
Unfortunately, they pressed a button and the screen went away, so I could not take a screenshot. Can maybe somebody nevertheless confirm this please? Is it a bug or on intention? If the latter, then maybe it could be made clearer? Thanks!

**Comment by Alayan-stk-2** [comment]
I think this is related to explosions, as you gain 10 points for a flag capture and 1 points when exploding another player. So you can have a tie in flag captures while having a difference in points.

It's an edge case.

**Comment by 7wells** [comment]
Yes, it's like you have described. The question is whether there is a winner in such a situation. If there is then s/he should clearly understand who won. Currently, there seem to be 2 contradictory messages. Maybe there could really be a tie with the next release, if points are equal, regardless where they come from. 😉

**Comment by Benau** [comment]
There is no 'winner' or 'Winner' string in stk (as in shown in gui based on supertuxkart.pot) so can't investigate until a screenshot is shown....
